bokeh/models/annotations/labels.py

```diff
@@ -77,6 +78,10 @@ class TextAnnotation(Annotation):
 
     '''
 
+    text = NonNullable(TextLike, default="", help="""
```

Comment:
Using `NonNullable` makes only if a default value is not provided.

Reply:
Yes, but how can I reuse TextLike here? TextLike does not default to empty string and I can't call it again here

Reply:
It would be good to make `TextLike` a proper property class. Then you would be able to instantiate it here with a default value and help. See e.g. `HatchPatternType` for how to implement this.